tests/functional_tests/ffront_tests/test_foast_parsing.py

```diff
@@ -282,3 +282,17 @@ def operator_with_externals(inp: Field[..., "float64"], inp2: Field[..., "float3
         kind=common_types.ScalarKind.FLOAT32, shape=None
     )
     assert "ext_unused" not in parsed.annex.symtable
+
+
+def test_set_values_at_call():
+    ADim = Dimension("ADim")
+    BDim = Dimension("BDim")
+
+    def bad_dim_where(a: Field[[ADim], float64]) -> Field[[BDim], float64]:
```

Comment:
```suggestion
    def wrong_return_type_annotation(a: Field[[ADim], float64]) -> Field[[BDim], float64]:
```
I don't care if it is "bad" or "wrong", but we used "wrong" in the rest of file.